src/main/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtils.java

```diff
@@ -0,0 +1,607 @@
+
+@SuppressWarnings("unused")
+public class ApacheCommonsStringUtils {
+
+    private static class Abbreviate {
+        @BeforeTemplate
+        String before(String s1, int width) {
+            return StringUtils.abbreviate(s1, width);
+        }
+
+        @AfterTemplate
+        String after(String s, int width) {
+            return s.substring(0, Math.min(s.length(), width));
+        }
+    }
+
+    private static class Capitalize {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.capitalize(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.substring(0, 1).toUpperCase() + s.substring(1);
+        }
+    }
+
+    //TODO: The test for this recipe fails, I think it could be due to a `rewrite-templating` bug
+    private static class Chomp {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.chomp(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.endsWith("\n") ? s.substring(0, s.length() - 1) : s;
+        }
+    }
+    private static class Chop {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.chop(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.substring(0, s.length() - 1);
+        }
+    }
+
+    private static class Contains {
+        @BeforeTemplate
+        boolean before(String s, String search) {
+            return StringUtils.contains(s, search);
+        }
+
+        @AfterTemplate
+        boolean after(String s, String search) {
+            return s.contains(search);
+        }
+    }
+
+    private static class CountMatches {
+        @BeforeTemplate
+        int before(String s, String pattern) {
+            return StringUtils.countMatches(s, pattern);
+        }
+
+        @AfterTemplate
+        int after(String s, String pattern) {
+            return s.length() - s.replace(pattern, "").length();
+        }
+    }
+
+    private static class DefaultString {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.defaultString(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return Objects.toString(s);
+        }
+    }
+
+    private static class DeleteWhitespace {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.deleteWhitespace(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.replaceAll("\\s+", "");
+        }
+    }
+
+    private static class EndsWithIgnoreCase {
+        @BeforeTemplate
+        boolean before(String s, String suffix) {
+            return StringUtils.endsWithIgnoreCase(s, suffix);
+        }
+
+        @AfterTemplate
+        boolean after(String s, String suffix) {
+            return s.regionMatches(true, 0, suffix, 0, suffix.length());
```

Comment:
This doesn't yet account for nulls, but should otherwise be correct:
```suggestion
            return s.regionMatches(true, s.length() - suffix.length(), suffix, 0, suffix.length());
```